kube_scheduler/datadog_checks/kube_scheduler/kube_scheduler.py

```diff
@@ -107,6 +107,10 @@
     'scheduler_pending_pods': 'pending_pods'
 }
 
+NEW_1_26_GAUGES = {
+    'scheduler_goroutines': 'scheduler_goroutines',
```

Comment:
We already have a metrics for kube_scheduler goroutines.
So usually we remap the new input metrics name to the exiting datadog metrics name. 

We do it if the 2 metrics expose the same information and same unit. Sometime a metric need a "transformer" function. because they changed the unit precision (for example from millicore to nanocore) 

the main advantage is that for user, it will be a seamless migration went they will update their kubernetes cluster version. and we don't need to update monitor and dashboard

 ```suggestion
    'scheduler_goroutines': 'goroutines',
```

Reply:
Based on discussion, this metric was renamed to `kube_scheduler.goroutine_by_scheduling_operation` to avoid confusion with the existing metric `kube_scheduler.goroutines`